terraform/helm/aptos-node/templates/networkpolicy.yaml

```diff
@@ -44,6 +44,19 @@ spec:
     ports:
     - protocol: TCP
       port: 9101
+  # Node Health Checker accesses these ports
+  - from:
+    - namespaceSelector: {}
+    - podSelector:
+        matchLabels:
+          app.kubernetes.io/name: node-health-checker
+    ports:
+    - protocol: TCP
+      port: 9101
+    - protocol: TCP
+      port: 8080
+    - protocol: TCP
+      port: 6180
```

Comment:
what type of check is running on 6180?

Reply:
@banool ? We require the node's noise port in configuration, but I'm not sure what checks are actually using that: https://github.com/aptos-labs/aptos-core/blob/b183a232784e4c77991b23e8728d3c7669a95d47/ecosystem/node-checker/configuration_examples/single_node_validator.yaml

Reply:
Yeah they're not, but that's still a potential development route, where we do noise based checks.